packages/discord.js/typings/index.d.ts

```diff
@@ -182,6 +182,14 @@ import {
   RawWidgetMemberData,
 } from './rawDataTypes';
 
+declare module 'node:events' {
+  class EventEmitter {
```

Comment:
If you are changing types for `node:events` it'll change for user's `EventEmitter` too. Is it intentional?

Reply:
Yes, it's overloaded with the djs client type as the event emitter as the first param. It has no effect on the event emitter types for anything else and still supports custom client events.